services/service.ts

```diff
@@ -52,22 +52,57 @@ export function mapFailure<S = any, F = any, R = any>(
 }
 
 export type PromiseRemoteDataResultMap<T, F> = { [P in keyof T]: Promise<RemoteDataResult<T[P], F>> };
+export type RemoteDataResultMap<T, F> = { [P in keyof T]: RemoteDataResult<T[P], F> };
+
+function createKeysMapTransformer<K = any>(keys: Array<K>) {
```

Comment:
As we use lodash, let's simplify it with `return (values) => _.fromPairs(_.zip(keys, values))`

Reply:
I plan to get rid of lodash